examples/bank/bank.go

```diff
@@ -118,7 +118,7 @@ func (bank *Bank) readAllAccounts() []int64 {
 		}
 		// Copy responses into balances.
 		for i := 0; i < bank.numAccounts; i++ {
-			if value, err := readInt64(calls[i]); err != nil {
+			if value, err := readInt64(calls[i].(client.GetCall)); err != nil {
```

Comment:
@tschottdorf this is an example of a case where batches get a little uglier. The slice needs to be exactly `[]Callable` when it is passed to Run, so we need to type-assert the call we take out. It's not much worse (we add a type assertion here but remove one in readInt64), but it's unfortunate that in go passing a slice to a variadic function requires an exact match of the type instead of a slice of any type that implements the right interface.